views/includes/inputs/fields/is-on-register-in-country-formed-input.html

```diff
@@ -1,29 +1,26 @@
+{% if title == "Tell us about the overseas entity" %}
```

Comment:
rather than have this include work out the hint text based on the title of the including page, it shoul d be possible to set hintText in the including page before you call this includes

eg
`{% set registerInCountryFormedText = "Is the other legal entity already on a public register in the country it was formed in?" %}
        {% set registrationNumberText = "Entity's registration number" %}
        {% set hintText = "Include the full name. For example, Jersey Financial Services Commission." %}
        {% include "includes/inputs/fields/is-on-register-in-country-formed-input.html" %}`